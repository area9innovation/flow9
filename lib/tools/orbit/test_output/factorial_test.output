Factorial of 5: 120
Result: "Factorial of 5: 120"
