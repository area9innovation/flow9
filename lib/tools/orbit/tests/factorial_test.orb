// Simple factorial test

// Define a factorial function
fn factorial(n) = n is (
  0 => 1;
  1 => 1;
  n => n * factorial(n - 1)
);

// Calculate some factorials
//println("Factorial of 0: " + i2s(factorial(0)));
//println("Factorial of 1: " + i2s(factorial(1)));
println("Factorial of 5: " + i2s(factorial(5)));