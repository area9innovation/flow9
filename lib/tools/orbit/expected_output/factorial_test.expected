Factorial of 5: 120
Result: 0
